entity-framework/ef6/fundamentals/relationships.md

```diff
@@ -148,7 +148,7 @@ In Entity Framework you use most commonly use the navigation properties to load
 
 ``` csharp
     // Get the course where currently DepartmentID = 1.
-    Course course2 = context.Courses.First(c=>c.DepartmentID == 2);
```

Comment:
Should it be 1 or 2? Variable is named course2.

Reply:
oh! ok, Then I change comment and it should be 2.